assets/js/modules/analytics/dashboard/dashboard-widget-all-traffic.js

```diff
@@ -61,6 +62,30 @@ class AnalyticsAllTraffic extends Component {
 		} );
 	}
 
+	getErrorDataComponent() {
+		const {
+			active,
+			setupComplete,
+		} = googlesitekit.modules.analytics;
+
+		const {
+			error,
+			receivingData,
+		} = this.state;
+
+		let errorDataComponent = null;
+
+		if ( active && ! setupComplete ) {
+			errorDataComponent = getSetupIncompleteComponent( __( 'Analytics', 'google-site-kit' ), true, true, true );
+		} else {
+			errorDataComponent = ! receivingData && (
+				error ? getDataErrorComponent( __( 'Analytics', 'google-site-kit' ), error, true, true, true ) : getNoDataComponent( __( 'Analytics', 'google-site-kit' ), true, true, true )
+			);
```

Comment:
For readability sake, can you simplify this to not be so "inline"?

I'd actually get rid of the `let errorDataComponent...` and simply have `if` clauses that immediately return the result of the call. Cleaner than having `if ... else`s.

Reply:
addressed https://github.com/google/site-kit-wp/pull/84/commits/faf30789b01dde54756154242ea3ff9792bdefeb